benchmark/torch/RL4LMs/rl4lms_utils/huggingface_generation_util.py

```diff
@@ -0,0 +1,1208 @@
+#   Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Remove the copyright declare here. This file is copied from the huggingface repo. Refer to the head lines in this file:
https://github.com/PaddlePaddle/PARL/blob/369da17e23812131d963c2e816d340ba2fc089fa/examples/ES/noise.py#L1